ManageWikiSettings.php

```diff
@@ -2837,43 +2846,61 @@
 		'type' => 'check',
 		'overridedefault' => false,
 		'section' => 'styling',
-		'help' => 'Disables the right rail that appears on appears on all pages (except Special Pages and MediaWiki Pages).',
+		'help' => 'Disables the right rail that appears on appears on all pages (except Special Pages, File Pages, Main Page and MediaWiki Pages).',
 		'requires' => [],
 	],
-	'wgEvelutionForceOneHeader' => [
-		'name' => 'Always visible Evelution Desktop Community Header',
+	'wgEvelutionServerMode' => [
+		'name' => 'Change Evelution layout to be more server/development wiki-friendly',
 		'from' => 'evelution',
 		'type' => 'check',
 		'overridedefault' => false,
 		'section' => 'styling',
-		'help' => 'On screens with 705 pixels or more, disables the desktop community header and forces the desktop sticky community header to be always visible.',
+		'help' => 'Changes the layout to be more server/development wiki-friendly (e.g. No Desktop Local Navigation, Addition of Left actions link before the content (which has the local navigation menus), smaller page header title, full width Small Breakpoint size, and no right rail on Help and Project namespaces).',
 		'requires' => [],
 	],
-	'wgEvelutionLeftPersonalLinks' => [
-		'name' => 'Move Evelution alignment of .upper-links container in the personal links to the left.',
+	'wgEvelutionStickyRail' => [
+		'name' => 'Disable Evelution Sticky Rail',
+		'from' => 'evelution',
+		'type' => 'check',
+		'overridedefault' => true,
+		'section' => 'styling',
+		'help' => 'Removes the sticky behavior of Right Rail on larger screens with 700 pixels or more. Requires <code>$wgEvelutionDisableRightRail</code> to not be set to work.',
+		'requires' => [],
+	],
+	'wgEvelutionForceFullWidth' => [
+		'name' => 'Force Evelution layout to be always full width',
 		'from' => 'evelution',
 		'type' => 'check',
 		'overridedefault' => false,
 		'section' => 'styling',
-		'help' => 'Change the alignment of the .upper-links container in the personal links from <code>center</code> to <code>left</code>.',
+		'help' => 'Makes content area being always at its maximum width, and cannot be toggled back to its original width.',
 		'requires' => [],
 	],
-	'wgEvelutionServerMode' => [
-		'name' => 'Change Evelution layout to be more server/development wiki-friendly',
+	'wgEvelutionDisableRightRailFromSpecificPages' => [
+		'name' => 'Disable Evelution Right Rail from specific pages',
+		'from' => 'evelution',
+		'type' => 'wikipages',
```

Comment:
What should I do instead?

Reply:
* Change the variable name to prefixed $wmg, and add a hack like #4546 into Defines.php.

Reply:
Though I'd prefer we just wait to do this until my fix is deployed to ManageWiki.

Reply:
If you want you can just remove that one config from ManageWikiSettings.php for now, so the rest can be done before that fix, as it may take a few days before I have much time to test it out and deploy it.

Reply:
> If you want you can just remove that one config from ManageWikiSettings.php for now, so the rest can be done before that fix, as it may take a few days before I have much time to test it out and deploy it.

Removed one. So the PR can now be merged

Reply:
This has now been fixed, so you can do another PR if you want now.

Reply:
> This has now been fixed, so you can do another PR if you want now.

#4574